tox.ini

```diff
@@ -1,31 +1,19 @@
 [tox]
 envlist = py{39,310,311,312,313},ruff,mypy
-isolated_build = true
+skipsdits = True
 
 [testenv]
-allowlist_externals = coverage
-skip_install = true
+runner = uv-venv-lock-runner
+dependency_groups = test
 commands =
-    coverage run
-    coverage xml
-    coverage report
+  {envpython} -m pytest -n auto --cov=gokart --cov-report=xml test -vv
```

Comment:
I think it is better to write `-n auto` at  `pytest.ini_options` in pyproject.toml.

Reply:
fixed in [41af399](https://github.com/m3dev/gokart/pull/429/commits/41af3999db3881094dec16e1f6a8b28eb378ecc6)